### ISSUE TITLE
 Minor modification for creating cards with Memento (an excellent video player based on MPV)

### ISSUE THREAD

**Issue by wandero** [yomidevs/local-audio-yomichan#28]
When using Local Audio Server in conjunction with Memento  (an excellent video player based on MPV) for card creation, the service was ineffective due to the lack of the Content-length parameter.
 
Following guidance from @ripose-jp, I modified the `server.py` code to resolve the issue and hope this modification can be incorporated into the plugin to benefit more users:

Added `import os`
#after self.send_header("Content-type", mime_type) Appended the line 
`self.send_header('Content-length', str(os.stat(audio_file).st_size))`

The plugin is very useful, thanks a lot

**Comment by Kuuuube** [comment]
Closing this since this is really an issue with memento. But if this is still something you wolud like to see, feel free to open a PR.

**Comment by ripose-jp** [comment]
It's not really an issue with Memento. I rely on Qt's Network library to handle HTTP requests for me and unfortunately it's written to assume that HTTP replies with the `Content-Length` header missing have no body. While it's not forbidden in the HTTP standard to not include a `Content-Length` header, it is unusual. I would have fixed this on my end years ago if I could.

**Comment by Kuuuube** [comment]
Using qt for networking seems... strange.

Regardless, I closed this before noticing that the change mentioned here appears to have been implemented already.

**Comment by ripose-jp** [comment]
>Using qt for networking seems... strange.

Not at all. The C++ standard library doesn't address networking at all currently, so if you're already using Qt there is little reason not to use their networking library as well. It integrates into the event loop, is portable, and doesn't require pulling in additional dependencies. It's not perfect of course, but it's good enough in most cases.

It looks like this was solved, it 2d7ddcefc941bb6cedaec82f395fdda460ccab83 though.
